Storage/Site.swift

```diff
@@ -36,6 +36,11 @@ open class Site: Identifiable {
     open var tileURL: URL {
         return URL(string: url)?.domainURL ?? URL(string: "about:blank")!
     }
+    
+    // i.e. `http://www.example.com/` --> `example`
+    open var secondLevelDomain: String? {
```

Comment:
if the domain is `domain.co.uk` this will return `co`. We have some helper functions here that might help :) https://github.com/mozilla-mobile/firefox-ios/blob/928caceddc8273c1f9d4885a57191e3ea70b81e8/Shared/Extensions/URLExtensions.swift#L152